templates/zerver/help/verify-that-your-message-has-been-successfully-sent.md

```diff
@@ -37,5 +37,5 @@ timestamp on the right side of the message body.
 ![Message time](/static/images/help/message-exact-time.png)
 
 Zulip is designed to store locally echoed message content in local
-storage and replay it when your browser reconnects to the Internet, to
-avoid issues where messages users thought they sent are lost.
+storage and replay it when your browser reconnects to the Internet, this is to
+avoid issues where messages, users thought they had sent get lost.
```

Comment:
The old grammar was correct but awkward; the new grammar is I think wrong (that comma shouldn't be there).